tests/test_linters.py

```
@@ -0,0 +1,26 @@
+            "--enable=E",
+            "make_profiler/lint_makefile.py",
+        ],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        text=True,
+    )
+    assert result.returncode == 0, result.stdout
+
+
+def test_ruff_runs_clean() -> None:
+    result = subprocess.run(
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)